includes/modules/server/Server.hpp

```diff
@@ -5,27 +5,33 @@
 
 #include "api/User.hpp"
 #include "api/Connection.hpp"
+#include <unistd.h>
 
 class Server : public CommandSender {
 private:
 	string name;
-	map<int, Connection *> connections;
+	vector<Connection *> connections;
+	struct sockaddr_in connectionConfig;
+	const string port;
+	const string password;
 
-	Connection *simulate_connect(int socket);
-	User *simulate_join(Connection *connection, string name);
+	void incomingConnection();
+	void incomingRequest(size_t index);
+	void closeConnection(size_t	index);
+	Connection *addConnection(const struct pollfd &connection);
 
 public:
-	Server(const string& name) : name(name) {}
+	Server(const string& name, const string& port, const string& password);
```

Comment:
Change `port` from `const string&` to `int` type.
```suggestion
	Server(const string& name, int port, const string& password);
```